File: online-judge-solutions/Codeforces/611A.cpp
//https://codeforces.com/problemset/problem/611/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int n;cin>>n;
	string a,s;cin>>a>>s;
	if(s=="month") cout<<(n<30?12:(n==30?11:7))<<endl;
	else cout<<((n==5 || n==6)?53:52)<<endl;
	return 0;
}

File: online-judge-solutions/Codeforces/611A.py
#https://codeforces.com/problemset/problem/611/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
a=input().split()
n,s=int(a[0]),a[2]
if(s=="month"): print(12 if n<30 else (11 if n==30 else 7))
else: print(53 if (n==5 or n==6) else 52)
